packages/haste-task-rollup/test/intex.spec.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
```

Comment:
I think you accidentally named this file `intex.spec.js`

Reply:
what name will be better?

Reply:
`index.spec.js`

Reply:
Ah😧
Yes. You are right. 
Thanks. Will fix